hal/src/main/native/include/hal/DriverStation.h

```diff
@@ -6,6 +6,8 @@
 
 #include <stdint.h>
 
+#include <cstddef.h>
```

Comment:
What compiler error do you get without this?

Reply:
Without this I get size_t is undefined

Reply:
Ok. Wel csstdef doesn't work here, because this needs to be compileable from C. stddef.h is better.

Reply:
That was what I was originally using but that caused a formatting fail, Will change